### DISCUSSION TITLE
School email verification

### DISCUSSION THREAD

**Discussion by sadasd526** [community/community#34796] [GitHub Education]
I dont get email from github to verify my school email, any ideas how to fix it?

**Comment by gowth6m** [comment]
Register with your university/school email or link your university/school email. 

1. In the upper-right corner of any page, click your profile photo, then click Settings.
2. In the "Access" section of the sidebar, click  Emails.
3. Under your email address, click Resend verification email.
4. GitHub will send you an email with a link in it. After you click that link, you'll be taken to your GitHub dashboard and see a confirmation banner.

https://docs.github.com/en/get-started/signing-up-for-github/verifying-your-email-address

I hope this helps with your problem.

**Reply by Birdza87** [reply]
โรงเรียนเพาะปัญญา
